mrtt-ui/package.json

```diff
@@ -3,6 +3,9 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@emotion/react": "^11.9.0",
+    "@emotion/styled": "^11.8.1",
+    "@mui/material": "^5.6.3",
```

Comment:
This works, but can we add [styled-components](https://styled-components.com/) here instead? @alanjleonard, myself and other Sparkgeo developers are more familiar with it.

More info on how to do this [here](https://mui.com/material-ui/guides/interoperability/#styled-components).

Reply:
yup, just removed emotion and replaced with the styled components package